tools/purge/signal.hh

```diff
@@ -69,11 +69,7 @@ typedef int bool;
 #endif
 #endif /* __cplusplus */
 
-#if 1 // so far, all systems I know use void
 # define SIGRETTYPE void
-#else
-# define SIGRETTYPE int
-#endif
```

Comment:
If we do this, then we should also remove `#if SIGRETTYPE != void` in the singal.cc file.

I would also remove SIGRETTYPE altogether and just use `void` then, but that is your call.

Reply:
Sounds good, doing that